Documentation/Contributors/CodingGuide/README.md

```diff
@@ -145,6 +152,14 @@ try {
 /*eslint-enable no-empty*/
 ```
 
+* If possible, use `//eslint-disable-line` to disable specific rules for one line.
+```js
+fn multiplyByThree(foo) {
```

Comment:
`fn` -> `function`, but I would instead try and find a real world case like

```javascript
window.alert(message); //eslint-disable-line no-alert`
```

Which we do in a few places to allow for alert.